doc-content/enterprise-only/admin-and-config/pessimistic-locking-cli-proc.adoc

```diff
@@ -0,0 +1,23 @@
+[id='pessimistic-locking-cli-proc_{context}']
+= Configuring pessimistic locking support using CLI mode
```

Comment:
I wouldn't call it a CLI mode as we don't really have a Command Line Interface. Can you rename it to just `Configuring pessimistic locking support programmatically`?